.drone.yml

```diff
@@ -75,11 +75,5 @@ pipeline:
 
 matrix:
   include:
-    - CORE_VERSION: 10.2.0
-      PHP_VERSION: 8.1
-    - CORE_VERSION: 10.2.0
-      PHP_VERSION: 8.2
-    - CORE_VERSION: 10.3.0
-      PHP_VERSION: 8.1
-    - CORE_VERSION: 10.3.0
-      PHP_VERSION: 8.2
+    - CORE_VERSION: 11
+      PHP_VERSION: 8.3
```

Comment:
This is not going to work! We need to keep support of D10 until June 2026. We can't force projects to go D11 all of a sudden. Also CAS 3.0 support D10, we should follow suit.